kubernetes/provider.go

```diff
@@ -101,6 +101,33 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("KUBE_LOAD_CONFIG_FILE", true),
 				Description: "Load local kubeconfig.",
 			},
+			"exec": {
+				Type:     schema.TypeSet,
```

Comment:
With `MaxItems: 1` configurations blocks, it is generally preferred to use `Type: schema.TypeList` to simplify handling. 👍